docs/mesos/ManualInstallCalicoMesos.md

```diff
@@ -0,0 +1,119 @@
+<!--- master only -->
+> ![warning](../images/warning.png) This document applies to the HEAD of the calico-docker source tree.
+>
+> View the calico-docker documentation for the latest release [here](https://github.com/projectcalico/calico-docker/blob/v0.13.0/README.md).
+<!--- else
+> You are viewing the calico-docker documentation for release **release**.
+<!--- end of master only -->
+
+# Manually Install Calico and Mesos
+This tutorial will walk you through installing Mesos, Netmodules, and Calico onto a Centos host to create a Mesos Slave compatible with Calico.
```

Comment:
You also need:
- an ETCD instance running and accessible
- Docker installed on your Centos host